linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_audispd_configure_remote_server/oval/shared.xml

```diff
@@ -1,10 +1,13 @@
+{{% if product in ["rhel8", "fedora", "ol8", "rhv4"] %}}
+{{% set audisp_config_file_path = "/etc/audit/audisp-remote.conf" %}}
+{{% else %}}
+{{% set audisp_config_file_path = "/etc/audisp/audisp-remote.conf" %}}
+{{% endif %}}
 <def-group>
   <definition class="compliance" id="auditd_audispd_configure_remote_server" version="1">
     <metadata>
       <title>Configure audispd Plugin Remote Server IP address or Hostname</title>
-      <affected family="unix">
-        <platform>multi_platform_all</platform>
-      </affected>
+      {{{- oval_affected(products) }}}
```

Comment:
The description just below would also benefit from the variable: `audisp_config_file_path`